packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
+   selectedCategories TEXT,
+   graphType TEXT DEFAULT 'BarGraph',
+   viewLegend INTEGER DEFAULT 0,
+   viewSummary INTEGER DEFAULT 0,
+   viewLabels INTEGER DEFAULT 0,
+   conditions TEXT,
+   conditions_op TEXT DEFAULT 'and',
+   sort_order REAL,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the table schema has a mixture of `camelCase` and `snake_case`. We should stick with just one style.